aws/table_aws_sagemaker_domain.go

```diff
@@ -0,0 +1,304 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsSageMakerDomain(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_sagemaker_domain",
+		Description: "AWS Sagemaker Domain",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"ValidationException", "NotFoundException", "ResourceNotFound"}),
+			Hydrate:           getAwsSageMakerDomain,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsSageMakerDomains,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The domain ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DomainId"),
+			},
+			{
+				Name:        "name",
+				Description: "The domain name.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DomainName"),
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the domain.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DomainArn"),
+			},
+			{
+				Name:        "creation_time",
+				Description: "A timestamp that indicates when the domain was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+
+			{
+				Name:        "app_network_access_type",
+				Description: "Specifies the VPC used for non-EFS traffic.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsSageMakerDomain,
+			},
+			{
+				Name:        "app_security_group_management",
+				Description: "The entity that creates and manages the required security groups for inter-app communication in VPCOnly mode.",
```

Comment:
```suggestion
				Description: "The entity that creates and manages the required security groups for inter-app communication in VPC Only mode.",
```
Should the description be changed as per the above suggestion?